geode-docs/tools_modules/redis_api_for_geode.html.md.erb

```diff
@@ -60,30 +65,47 @@ If the server is functioning properly, you should see a response of `PONG`.
 
 The Redis API for <%=vars.product_name%> currently supports the following commands.  See [Redis commands](https://redis.io/commands/) for a complete list of and more information on Redis commands.
 
--   **Strings**: APPEND, GET, GETRANGE, SET
+-   **Strings**: APPEND, GET, SET
 -   **Hashes**: HGETALL, HMSET, HSET
 -   **Sets**: SADD, SMEMBERS, SREM
 -   **Publish/Subscribe**: PUBLISH, PSUBSCRIBE, PUNSUBSCRIBE, SUBSCRIBE, UNSUBSCRIBE
--   **Keys**: DEL, EXISTS, EXPIRE, PERSIST, PEXPIRE, PEXPIREAT, RENAME
+-   **Keys**: DEL, EXISTS, EXPIRE, EXPIREAT, KEYS, PERSIST, PEXPIRE, PEXPIREAT, PTTL, RENAME, TTL, TYPE
+-   **Connection**: AUTH, PING, QUIT
 
-The following Redis API for <%=vars.product_name%> commands are **experimental**:
+The following Redis API for <%=vars.product_name%> commands are **unsupported**:
 
--   **Strings**: BITCOUNT, BITOP, BITPOS, DECR, DECRBY, GETBIT, GETSET, INCR, INCRBY, INCRBYFLOAT, MGET,
-  MSET, MSETNX, PSETEX, SETBIT, SETEX, SETNX, STRLEN
--   **Lists**: LINDEX, LLEN, LPOP, LPUSH, LPUSHX, LRANGE, LREM, LSET, LTRIM, RPOP, RPUSH, RPUSHX
--   **Hashes**: HDEL, HEXISTS, HGET, HINCRBY, HINCRBYFLOAT, HKEYS, HMGET, HSETNX, HLEN, HSCAN, HVALS
+-   **Strings**: BITCOUNT, BITOP, BITPOS, DECR, DECRBY, GETBIT, GETRANGE, GETSET, INCR, INCRBY, INCRBYFLOAT, MGET,
+  MSET, MSETNX, PSETEX, SETBIT, SETEX, SETNX, SETRANGE, STRLEN
+-   **Hashes**: HDEL, HEXISTS, HGET, HINCRBY, HINCRBYFLOAT, HKEYS, HLEN, HMGET, HSCAN, HSETNX, HVALS
 -   **Sets**: SCARD, SDIFF, SDIFFSTORE, SINTER, SINTERSTORE, SISMEMBER, SMOVE, SPOP, SRANDMEMBER,
   SSCAN, SUNION, SUNIONSTORE
--   **SortedSets**: ZADD, ZCARD, ZCOUNT, ZINCRBY, ZLEXCOUNT, ZRANGE, ZRANGEBYLEX, ZRANGEBYSCORE,
-  ZRANK, ZREM, ZREMRANGEBYLEX, ZREMRANGEBYRANK, ZREMRANGEBYSCORE, ZREVRANGE, ZREVRANGEBYSCORE,
-  ZREVRANK, ZSCAN, ZSCORE
--   **HyperLogLog**: PFADD, PFCOUNT, PFMERGE
--   **Keys**: DBSIZE, EXPIREAT, FLUSHDB, FLUSHALL, KEYS, PTTL, SCAN, TTL
--   **Transactions**: DISCARD, EXEC, MULTI
--   **Server**: AUTH, ECHO, PING, QUIT
+-   **Keys**: FLUSHALL (no async option), FLUSHDB (no async option), SCAN
+-   **Server**: DBSIZE, ECHO, TIME, SHUTDOWN
```

Comment:
DBSIZE makes more sense to be in the "Keys" section since it tells you how many keys exist

Reply:
Redis groups it with other Server commands: https://redis.io/commands#server
Should we use the same grouping as Redis?

Reply:
Just realizing we need to move FLUSHALL and FLUSHDB as well since Redis groups them with the Server commands

Reply:
I agree that we should follow redis's grouping. I had not even noticed that redis had the "filter by group" drop down.
So: FLUSHALL and FLUSHDB are also in the "Server" group. So that leaves just SCAN in our unsupported "Keys" group.

Reply:
I just reorganized them, please double check though.

Reply:
looks good